### PR TITLE
Add a few more top level catch-alls so that we don't miss on some PRs

### PR DESCRIPTION
**Pull request by brendandburns**
@thockin @smarterclayton @bgrant0607 @derekwaynecarr 

Moar owners for moar issuez

### CODE REVIEW

File: mungegithub/blunderbuss.yml [hunk 1]
```diff
@@ -118,6 +118,24 @@ prefixMap:
...
+    - zmerlynn
+    - brendandburns
+    - eparis
+
+  examples:
+    - bgrant0607
+    - thockin
+    - brendandburns
+
   hack/:
```
Comment (bgrant0607) [inline mungegithub/blunderbuss.yml]: Does it matter that hack ends in `/`, but other directories don't?

**Comment by bgrant0607** [comment]
LGTM

File: mungegithub/blunderbuss.yml (hunk 1 above)
Comment (brendandburns) [inline mungegithub/blunderbuss.yml]: only in that it confuses people.  fixed.

**Comment by eparis** [comment]
you can add me to hack if you want.

**Comment by eparis** [comment]
but LGTM

**Comment by brendandburns** [comment]
done, and merged on green.

**Pull request merged**